Lib9c/BlockPolicy.cs

```diff
@@ -60,7 +66,10 @@ public static IBlockPolicy<PolymorphicAction<ActionBase>> GetPolicy(int minimumD
 #endif
         }
 
-        public static ActionRenderer GetRenderer() => ActionRenderer;
+        public static IRenderer<NCAction> GetRenderer(bool actionRender) =>
```

Comment:
Returning it as an abstract type `IRenderer<T>` instead of a concrete type (`BlockRenderer` or `ActionRenderer`) makes client code difficult to use its concrete methods like `EveryBlock()` or `EveryRender()`…

How about making `BlockRenderer` and `ActionRenderer` fields `public`, instead of having `bool actionRender` flag?

Reply:
That would be better. Changed it.